src/components/organisms/NewProfileModal/components/ProfileModalQuestions/ProfileModalQuestions.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../profileModalStyles";
+
+$question-container-gap: 0.7rem;
+$question-answer-gap: 4px;
+$question-answer-gap-editable: 0.3rem;
+$question-color: rgba($white, 0.6);
```

Comment:
opaque-white

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/85b20b3dbb6ace046e5a2eadfd426df41e2fd9c5